generated/nodejs/google-cloud-node/packages/trace/package.json

```diff
@@ -11,10 +11,17 @@
   "dependencies": {
     "google-proto-files": "^0.8.2",
     "google-gax": "^0.7.0",
-    "extend": "^3.0.0"
+    "extend": "^3.0.0",
+    "google-auto-auth": "~0.2.4",
+    "google-proto-files": "^0.8.3",
+    "grpc": "~1.0",
+    "mocha": "3.2.0"
```

Comment:
Please add this as a `devDependency` so that normal users won't need to install mocha.